python/singa/tensor.py

```diff
@@ -87,10 +87,10 @@ class Tensor(object):
                      grad but not store grad; But if a tensor stores grad
                      then it must require grad.
     '''
-
+    tensor_count = 0
```

Comment:
where is this var used?

Reply:
`if name is None:
            self.name = 'Dummy#{}'.format(Tensor.tensor_count)
            Tensor.tensor_count += 1
        else:
            self.name = name`
add default name, we did not have self.name before, so i added it

Reply:
I imitated from autograd.py's operator name code